build.gradle

```diff
@@ -10,4 +10,29 @@ buildscript {
 subprojects {
     apply plugin: 'scalafmt'
     scalafmt.configFilePath = gradle.scalafmt.config
+
+    afterEvaluate {
+        if (project.plugins.hasPlugin('application')) {
+            startScripts {
+                doLast {
+                    unixScript.text = configureUnixClasspath(unixScript)
+                }
+            }
+        }
+    }
```

Comment:
I googled for a bit and stumbled upon:

```groovy
startScripts {
  classpath += files('ext-libs/*', 'config')
}
```

There seems to be a possibility to do this other than manipulating raw lines (which feels a bit rough). I'd need to look further into this, since gradle seems to apply some sanitizing unfortunately.

Reply:
Thanks @markusthoemmes  for looking into this. I also saw that but it does not work as expected. Have a look at links provided in #3057 for background and hence used this approach

Reply:
@chetanmeh oh sorry, should've checked. Yeah your observations are exactly the same as mine, so we're clear on that 👍